apps/landing-page/pages/blog/using-openshift-authentication-to-secure-access-to-backstage.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Using OpenShift Authentication to Secure Access to Backstage
+date: 2023/02/16
+description: Using OpenShift Authentication to Secure Access to Backstage
+---
+
+import { Authors } from 'ui/components';
+import { Callout } from 'nextra-theme-docs';
```

Comment:
This line can be removed because it looks like you don't use a `Callout` in this blog post.

Reply:
updated!